keyboards/40percentclub/mf68/keymaps/emdarcher/keymap.c

```diff
@@ -48,8 +48,6 @@ void led_set_user(uint8_t usb_led){
     } else {
         //set to Hi-Z
         setPinInput(B0);
-        writePinLow(B0);
-        setPinInput(D5);
-        writePinLow(D5);
+        setPinInput(B5);
```

Comment:
Why B5?
I think it will be D5. Is it true?

Reply:
Yes, it should be D5, not B5 as this is using the two TX and RX LEDs on the Pro Micro

Reply:
The `setPinInput(D5)` should be kept, not changed to `setPinInput(B5)`.